lib/sorbet-rails/railtie.rb

```diff
@@ -15,5 +15,13 @@ class SorbetRails::Railtie < Rails::Railtie
       ActiveRecord::Base.extend SorbetRails::CustomFinderMethods
       ActiveRecord::Relation.include SorbetRails::CustomFinderMethods
     end
+
+    # in test & dev, the models are not pre-loaded, we need to load them manually
+    Rails.application.eager_load! unless Rails.env.production?
```

Comment:
I wonder if there is a better way to do this :-? eg we can trigger it on any subclass of ActiveRecord::Base automatically